deploy/k3s/inference_deployment/inference_deployment_template.yaml

```diff
@@ -40,7 +40,7 @@ spec:
           maxUnavailable: 0  # Aim for no downtime during rollout
       containers:
       - name: inference-server
-        image: 767397850842.dkr.ecr.us-west-2.amazonaws.com/gl-tritonserver-test:674426bc8-main
+        image: 767397850842.dkr.ecr.us-west-2.amazonaws.com/gl-tritonserver-test:latest
```

Comment:
The tradeoff with doing this is that now whenever we push a new commit that builds this image, we will have a new :latest image. And these images are (currently) large (10-15Gb?) and take a long long time to pull to the edge device. Not a big concern at the moment but good to keep in mind that this isnt a free lunch.

Reply:
What we really need here is a cicd pipeline matches all the versions together and some simpler tools for development. 

But I still think this is a step forward and we should do it.